cwms-data-api/src/main/java/cwms/cda/api/TimeSeriesGroupController.java

```diff
@@ -82,6 +82,10 @@ private Timer.Context markAndTime(String subject) {
                         + " the assigned timeseries in the returned timeseries groups. (default: true)"),
                 @OpenApiParam(name = TIMESERIES_CATEGORY_LIKE, description = "Posix <a href=\"regexp.html\">regular expression</a> "
                         + "matching against the timeseries category id"),
+                @OpenApiParam(name = CATEGORY_OFFICE_ID, description = "Specifies the owning office of the "
```

Comment:
I could use some input on the language for describing the different office parameters.

Reply:
looks good to me